client/src/Questions/QuestionsForm.js

```diff
@@ -0,0 +1,71 @@
+import { useState, useEffect } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import Api from "../../Api";
+
+const QuestionForm = () => {
+  const history = useHistory();
+  const { id } = useParams();
+  const [questions, setQuestions] = useState({
```

Comment:
This should really be singular question/setQuestion since it represents ONE question